web/satellite/src/components/account/billing/billingTabs/Overview.vue

```diff
@@ -1,22 +1,266 @@
+                    >
+                        View Billing History →
+                    </p>
+                </div>
+                <div class="total-cost__card">
+                    <AvailableBalanceIcon class="total-cost__card__main-icon"/>
+                    <p class="total-cost__card__money-text">{{ balance.coins | centsToDollars }}</p>
+                    <p class="total-cost__card__label-text">Available Balance</p>
+                    <p class="total-cost__card__link-text"
+                    @click="routeToPaymentMethods"
+                    >
+                        View Payment Methods →
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="cost-by-project">
+            <h3 class="cost-by-project__title">Cost by Project</h3>
+            <UsageAndChargesItem2
+                v-for="usageAndCharges in projectUsageAndCharges"
+                :key="usageAndCharges.projectId"
+                :item="usageAndCharges"
+                class="cost-by-project__item"
+            />
+        </div>
         <router-view />
     </div>
 </template>
 
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator';
+import { RouteConfig } from '@/router';
+
+import UsageAndChargesItem2 from '@/components/account/billing/estimatedCostsAndCredits/UsageAndChargesItem2.vue';
+
+import EstimatedChargesIcon from '@/../static/images/account/billing/totalEstimatedChargesIcon.svg';
+import AvailableBalanceIcon from '@/../static/images/account/billing/availableBalanceIcon.svg';
+import CalendarIcon from '@/../static/images/account/billing/calendar-icon.svg';
+
+import { AccountBalance } from '@/types/payments';
+import { ProjectUsageAndCharges } from '@/types/payments';
+import { PAYMENTS_ACTIONS } from '@/store/modules/payments';
+import { PROJECTS_ACTIONS } from '@/store/modules/projects';
+
 
 // @vue/component
 @Component({
     components: {
+        EstimatedChargesIcon,
+        AvailableBalanceIcon,
+        UsageAndChargesItem2,
+        CalendarIcon,
     },
 })
 export default class BillingArea extends Vue {
+    // public estimatedCharges: number = 0;
+    public availableBalance: number = 0;
+    public showChargesTooltip: boolean = false;
+    public isDataFetching = true;
+    public charges = this.projectUsageAndCharges;
+    public currentDate: string = ""
+
+
+
+    /**
+     * Lifecycle hook after initial render.
+     * Fetches projects and usage rollup.
+     */
+    public async mounted(): Promise<void> {
+        try {
+            await this.$store.dispatch(PROJECTS_ACTIONS.FETCH);
+        } catch (error) {
+            this.isDataFetching = false;
+            return;
+        }
+
+        try {
+            await this.$store.dispatch(PAYMENTS_ACTIONS.GET_PROJECT_USAGE_AND_CHARGES_CURRENT_ROLLUP);
+
+            this.isDataFetching = false;
+        } catch (error) {
+            await this.$notify.error(error.message);
```

Comment:
in case of error there is no release for isDataFetching flag

Reply:
added release